packages/client/test/rpc/engine/getPayloadV3.spec.ts

```diff
@@ -110,6 +110,16 @@ describe(method, () => {
     ).sign(pkey)
 
     await service.txPool.add(tx, true)
+
+    // check the blob and proof is available via getBlobsV1
+    res = await rpc.request('engine_getBlobsV1', [txVersionedHashes])
+    const blobsAndProofs = res.result
+    for (let i = 0; i < txVersionedHashes.length; i++) {
+      const { blob, proof } = blobsAndProofs[i]
+      assert.equal(blob, txBlobs[i])
+      assert.equal(proof, txProofs[i])
+    }
```

Comment:
I suggest that there should also be a test which tests the cache mechanism here.

Reply:
cache pruning you mean?

Reply:
Yes